### DISCUSSION TITLE
Does not see the video card

### DISCUSSION THREAD

**Discussion by beba1G** [w-okada/voice-changer#842] [Q&A]
I'm setting up the program everything works, but it's very crooked because I can't find a video card because it's not in the video card selection line, tell me how to solve the problem (video card if anything AMD rx 5500xt)
![image](https://github.com/w-okada/voice-changer/assets/144958790/2be16a69-edb8-4f44-957f-fe2053b8cb4d)

**Comment by Darkskys0** [comment]
For AMD you are gonna want to download the version which supports AMD (ONNX(cpu,DirectML). From what I am seeing you downloaded the Cuda version which is for Nvidia.
 Once you download the DirectML version, you will see options for CPU, GPU0, GPU1, GPU2, and GPU3. At this point open your task manager and go to the performance tab, then scroll down till you see your GPU. It will be labeled GPU-0 or higher and that will be the one you select in voice changer.
